### DISCUSSION TITLE
Roboshop-v3 execution error- Local-exec provision error

### DISCUSSION THREAD

**Discussion by maheshkoheda** [learndevopsonline/learndevopsonline#1524] [Q&A]
@r-devops Hi raghu, I am trying to execute roboshop-v3 but i am getting 
1) Local-exec provision error
2) null_resource.ansible is tainted and must be replaced
So the instances and Route53 are created but frontend not reachable.
Could you please check and let me know what is going wrong here.

<img width="938" alt="Roboshop-v3 error" src="https://github.com/learndevopsonline/learndevopsonline/assets/25144175/6e73c31c-473e-4866-be5d-b48f09445a14">

**Comment by sreedharm07** [comment]
hi @maheshkoheda
did error still exists ? if not can you please turn on your machine and share your ip

**Reply by maheshkoheda** [reply]
Hi @sreedharm07 yes the error still exists. I have turned on my workstation and attached the screenshot with ip details.
<img width="806" alt="workstation" src="https://github.com/learndevopsonline/learndevopsonline/assets/25144175/519a29bd-336a-4166-b458-8746e5006b10">

**Reply by sreedharm07** [reply]
hi @maheshkoheda 
your dns record name is **cart.mahe....**. but in your code you are calling **cart-dev.mahes...**.  thats why its not able to ssh please change dns names or your code.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/3e7cb1fa-7d65-4ce8-9b45-27d7fc902f1b)




![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/ef7dfddf-41b9-4fe6-967f-8d7c66460bb0)
